### DISCUSSION TITLE
White list of single IP address ignored in connectban

### DISCUSSION THREAD

**Discussion by tyctor** [inspircd/inspircd#2092] [Support]
Hi

I have monitoring server, which check if `inspircd` is running on remote `host:port`, it repeat checking every 1-2 minutes
I also have defined `connectban` module enabled in config, which causes that IP of monitoring server is banned.

Is there some way how to white list single IP address and prevent it from to be banned?

I am looking forward for answer

**Comment by sadiepowell** [comment]
You can [E-line the IP address that your monitoring server is connecting from](https://docs.inspircd.org/3/configuration/#exception) to exempt it from bans.

**Reply by tyctor** [reply]
perfect, thanks
